### PR TITLE
Celery5

### PR DESCRIPTION
**Pull request by dlbrittain**
Make templates compatible with celery5 materialization engine refactor.

### CODE REVIEW

**Review by fcollman — APPROVED** [review]

**Comment by fcollman** [comment]
the reccomended min version of materialization needs to be bumped to support these changes

**Comment by fcollman** [comment]
I meant to retract my original approve pending that comment.. can't figure out how to do it

**Comment by dlbrittain** [comment]
> the reccomended min version of materialization needs to be bumped to support these changes

Once we merge the celery5 PR into main on materialization this will be bumped to use the correct mat version

**Pull request merged**
